.golangci.yml

```diff
@@ -3,6 +3,22 @@ linters:
     - unconvert
     - unparam
     - revive
+    - misspell
```

Comment:
could you please also enable these checker for dm? https://github.com/pingcap/ticdc/blob/master/dm/.golangci.yml

Reply:
It's already included: https://github.com/pingcap/ticdc/blob/71e2a4818e91b8b7fc4ffe90a936f0772c36f2b9/dm/.golangci.yml#L18